src/modules/variable/init.rs

```diff
@@ -55,14 +55,16 @@ impl SyntaxModule<ParserMetadata> for VariableInit {
 impl TranslateModule for VariableInit {
     fn translate(&self, meta: &mut TranslateMetadata) -> String {
         let name = self.name.clone();
-        let mut  expr = self.expr.translate(meta);
+        let mut expr = self.expr.translate(meta);
         if let Type::Array(_) = self.expr.get_type() {
             expr = format!("({expr})");
         }
-        let local = if self.is_fun_ctx { "local " } else { "" };
-        match self.global_id {
-            Some(id) => format!("__{id}_{name}={expr}"),
-            None => format!("{local}{name}={expr}")
+        if let Some(id) = self.global_id {
```

Comment:
We previously tested `self.global_id` _after_ `self.is_fun_ctx`, but I switched the order to matches the shorthand operators.  However, I think it is not possible for both `global_id` and `is_fun_ctx` to be enabled at the same time.